frame/clearing-house/src/lib.rs

```diff
@@ -134,20 +136,34 @@ pub mod pallet {
 			+ Clone
 			+ PartialEq
 			+ Debug;
+
 		/// Signed decimal fixed point number.
 		type Decimal: FullCodec + MaxEncodedLen + TypeInfo + FixedPointNumber;
+
 		/// Implementation for querying the current Unix timestamp
 		type UnixTime: UnixTime;
+
 		/// Virtual Automated Market Maker pallet implementation
-		type Vamm: Vamm<Balance = Self::Balance, Decimal = Self::Decimal>;
+		type Vamm: Vamm<
+			Balance = Self::Balance,
+			Decimal = Self::Decimal,
+			VammConfig = Self::VammConfig,
+		>;
+
+		/// Configuration for creating and initializing a new vAMM instance. To be used as an
+		/// extrinsic input
+		type VammConfig: FullCodec + MaxEncodedLen + TypeInfo + Debug + Clone + PartialEq;
+
 		/// Price feed (in USDT) Oracle pallet implementation
```

Comment:
I think I did that because of the API documentation of the `composable-traits::oracle::Oracle` trait:
```rust
/// An object that is able to provide an asset price.
/// Important: the current price-feed is providing prices in USDT only.
pub trait Oracle {
```

Reply:
Oh I see. 
Alright then :+1:

Reply:
Your comment makes sense though, I'll keep that in mind. Perhaps the Oracle will change in the future. Besides, it is weird that this doc is in the trait declaration instead of the actual implementation of it in `pallet-oracle`

Reply:
Yep. Thought the same.